docs/maintenance/Pull_Requests.mdx

```diff
@@ -24,15 +24,105 @@ Per the scales from [Contribution Tiers > Pointing](./Contributor_Tiers.mdx#poin
+
+Add the `awaiting response` label to a PR whenever you post a review.
+It will be automatically removed if the author re-requests review.
+
+### Be Kind
+
+Above all else, be _encouraging_ and _friendly_ in tone.
+
+- Phrase feedback as opportunities for improvement rather than chastising.
+- Call out any positive aspects you see - especially in larger pull requests.
+- Use happy emojis frequently.
+- If you have the energy, post a fun (but not inappropriate) GIF with your review.
+  - We often use the _GIFs for Github_ extension: available as [GIFs for GitHub (Chrome)](https://chrome.google.com/webstore/detail/gifs-for-github/dkgjnpbipbdaoaadbdhpiokaemhlphep) and [GIFs for GitHub (Firefox)](https://addons.mozilla.org/en-US/firefox/addon/gifs-for-github).
+
+Pull requests are the first time many community members meaningfully interact with us - or, oftentimes, any open source maintainers at all.
+It's important that we provide a positive, uplifting experience. ❤️
+
+### Reviewing a PR
+
+Before reviewing a pull request, try to look back over the backing issue to (re-)familiarize yourself with it.
+You may find it useful to:
+
+- Attempt to simplify the provided reduction (["code golf"](https://en.wikipedia.org/wiki/Code_golf))
+- Look back through previous issues and PRs around the same area of code / lint rule
+
+If there's no backing issue:
+
+- If the PR is a straightforward docs or tooling fix that doesn't impact users, it's ok to review it as-is
+- Otherwise, add the `please fill out the template` label and post a comment like the _Needs Issue_ template
+
+Upon completing your review, if the build is passing and you feel comfortable merging it in, go ahead and squash merge.
+Otherwise, add the `1 approval` label.
+
+#### Common Things to Look For
+
+- Style: that can you read through the code easily, there are comments for necessary tricky areas, and not unnecessary comments otherwise.
+  - Try not to nitpick things that don't matter.
+  - If something is standard in the repo but not enforced, consider mentioning it as a non-blocking comment and filing an issue to enforce it.
+- Thoroughness: does it handle relevant edge cases? Commonly:
+  - Generics and type parameters (see: `getConstrainedTypeAtLocation`).
+  - Parenthesis and whitespace (see: `getWrappingFixer`).
+  - Unions and intersections (see: `unionTypeParts` and `intersectionTypeParts`).
+- Unit tests:
+  - All lines are covered per the Codecov / `yarn jest path/to/impacted/file --coverage` report.
+  - Both "positive" and "negative" ("valid" and "invalid") cases exist, if reasonably possible to test for.
+  - Fixes and suggestions, if present, don't remove `//` or `/*` comments
+  - `invalid` lint rule errors include line and column information
+  - Valid syntax edge cases don't cause the rule to crash, even if they cause a type error in TypeScript
```

Comment:
FYI @auvred, added a couple here since your approval (see linked PRs).

Reply:
Thanks for pointing! This sounds great!